db/migrate/20190509223219_add_profile_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddProfileToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :users, :profile, foreign_key: true
```

Comment:
IMO, this should be:
```
add_reference :profiles, :users, foreign_key: true
```

Reply:
Replied with our logic above. Waiting for guidance.